src/app/core/components/left-header/left-header.component.html

```diff
@@ -12,6 +11,6 @@
 
 <div class="nav-control alt" *ngIf="compactMode">
   <span class="nav-collapse" (click)="showLeftMenuAndHeaders()" tabindex="0">
-    <img src="assets/images/menu-open.svg" style="width: 2rem; height: 2rem;" />
+    <i class="fas fa-compress-arrows-alt" style="font-size: 2rem;"></i>
```

Comment:
This one looks a bit like the sign for emergency rendez-vous point. ;-)

Reply:
Changed icon.